File: tests/testdata/codegen/output-literals-pp/dotnet/output-literals.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;

return await Deployment.RunAsync(() => 
{
    return new Dictionary<string, object?>
    {
        ["output_true"] = true,
        ["output_false"] = false,
        ["output_number"] = 4,
        ["output_string"] = "hello",
    };
});

File: pkg/codegen/testing/utils/testdata/output-literals-pp/go/output-literals.go
package main

import (
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		ctx.Export("output_true", pulumi.Bool(true))
		ctx.Export("output_false", pulumi.Bool(false))
		ctx.Export("output_number", pulumi.Float64(4))
		ctx.Export("output_string", pulumi.String("hello"))
		return nil
	})
}
